scripts/bigquery_billing_report/bigquery_billing_report.py

```diff
@@ -317,16 +315,27 @@ def main():
     parser = DefaultArgsParser(description="Print a report about costs for a given submission.")
 
     # Core application arguments
-    parser.add_argument('-p', '--namespace', dest='ws_namespace', action='store', required=True, help='Workspace namespace')
+    parser.add_argument('-p', '--namespace', dest='ws_namespace', action='store', required=True,
+                        help='Workspace namespace')
     parser.add_argument('-n', '--name', dest='ws_name', action='store', required=True, help='Workspace name')
 
-    parser.add_argument('-s', '--subid', dest='submission_id', action='store', required=False, help='Optional Submission ID to limit the pricing breakdown to.  If not provided pricing for all submissions in this workspace will be reported.')
-    parser.add_argument('-w', '--wfid', dest='workflow_id', default=[], action='append', required=False, help='Optional Workflow ID to limit the pricing breakdown to.  Note that this requires a submission id to be passed as well."')
+    parser.add_argument('-s', '--subid', dest='submission_id', action='store', required=False,
+                        help='Optional Submission ID to limit the pricing breakdown to.  If not provided pricing for all submissions in this workspace will be reported.')
+    parser.add_argument('-w', '--wfid', dest='workflow_id', default=[], action='append', required=False,
+                        help='Optional Workflow ID to limit the pricing breakdown to.  Note that this requires a submission id to be passed as well."')
 
-    parser.add_argument('-c', '--calls', dest='show_all_calls', action='store_true', required=False, help='Expand information about each call.')
+    parser.add_argument('-c', '--calls', dest='show_all_calls', action='store_true', required=False,
+                        help='Expand information about each call.')
 
-    parser.add_argument('-dp', '--dataset_project', dest='dataset_project', action='store', required=False, help='Optional project where dataset is stored - defaults to the workspace project.')
-    parser.add_argument('-dn', '--dataset_name', dest='dataset_name', action='store', required=False, help='Optional dataset name where billing data is stored - defaults to billing_export.')
+    parser.add_argument('-bp', '--bill_to_project', dest='bill_to_project', action='store', required=False,
```

Comment:
choose where to run the query - you need to provide a project you are authorized to run/bill BQ queries to